Freedom wall

Tapispisan-YvesJohn opened the pull request:
fix ui and add FormsModule

Tapispisan-YvesJohn committed:
add freedom wall files

Tapispisan-YvesJohn committed:
fix freedom wall

Tapispisan-YvesJohn committed:
try fix modal

Villa-VillaDiana committed:
update table and change file location of service

Villa-VillaDiana committed:
Merge branch 'login-and-signup' of https://github.com/PUP-BSIT/project-hypefive_appdev into login-and-signup

Villa-VillaDiana committed:
update login.component.ts

Villa-VillaDiana committed:
remove duplicate lines

Villa-VillaDiana committed:
add seeder

Villa-VillaDiana committed:
update response

Villa-VillaDiana committed:
Merge pull request #8 from PUP-BSIT/main

pull from main

Villa-VillaDiana committed:
update the backend for signup/registration

Tapispisan-YvesJohn committed:
fix styling

Villa-VillaDiana committed:
update table and controller

Villa-VillaDiana committed:
Merge pull request #10 from PUP-BSIT/main

Pull main changes

Ternal-AprilJoy commented on orgbee/src/app/pages/freedom-wall/freedom-wall.component.css:
```diff
@@ -0,0 +1,172 @@
+  background-color: gray;
+}
+
+
```
Remove excess space.

Ternal-AprilJoy commented on orgbee/src/app/pages/freedom-wall/freedom-wall.component.html:
```diff
@@ -1 +1,36 @@
+</div>
+
+<ngx-masonry class="masonry-container">
+  <ngx-masonry-item *ngFor="let post of posts">
```
Use @for instead.

Ternal-AprilJoy commented on orgbee/src/app/pages/freedom-wall/freedom-wall.component.html:
```diff
@@ -1 +1,36 @@
+
+<ngx-masonry class="masonry-container">
+  <ngx-masonry-item *ngFor="let post of posts">
+    <div class="masonry-item" [ngStyle]="{'background-color': post.color}" 
```
Never use NGstyle put it in css instead.

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_0

Ternal-AprilJoy commented on orgbee/src/app/pages/freedom-wall/freedom-wall.component.ts:
```diff
@@ -1,10 +1,49 @@
   styleUrl: './freedom-wall.component.css'
 })
 export class FreedomWallComponent {
+  posts: any[] = [];
+  newPostText: string = '';
+  showPostCreation: boolean = false;
+  showModal: boolean = false;
```
Dont use any type. I think you can write it like this.


```suggestion
   posts: Post[] = [];
  newPostText: string = '';
  showPostCreation: boolean = false;
  showModal: boolean = false;
```

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g26feb3ebb2b_0_0

Ternal-AprilJoy commented on orgbee/src/app/post-dialog/post-dialog.component.html:
```diff
@@ -0,0 +1,4 @@
+<div class="dialog-content" [ngStyle]="{ 'background-color': data.backgroundColor }">
```
Dont use ngStyle. Kindly write your styling in the css file.

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_0

Ternal-AprilJoy commented on orgbee/src/app/post-dialog/post-dialog.component.ts:
```diff
@@ -0,0 +1,21 @@
+  styleUrls: ['./post-dialog.component.css']
+})
+export class PostDialogComponent {
+  constructor(@Inject(MAT_DIALOG_DATA) public data: any, private elementRef: ElementRef){}
```
Kindly reconstruct this code and not use 'any' type.
reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2c15720ff99_0_64

Villa-VillaDiana committed:
update signup

Villa-VillaDiana committed:
add announcements table

Villa-VillaDiana committed:
add tostr

Villa-VillaDiana committed:
update login and routing

Ternal-AprilJoy committed:
create login service.

Ternal-AprilJoy committed:
fix routing.

Villa-VillaDiana committed:
add files from dist to public

Villa-VillaDiana committed:
update web routing

Villa-VillaDiana committed:
update routing

Villa-VillaDiana committed:
update routing

MentokAlmera committed:
modify member page

MentokAlmera committed:
fix spacing

Tapispisan-YvesJohn committed:
fix issues with the code review

Tapispisan-YvesJohn commented:
I apologize for the delay. I will submit the pull request after finalizing my code. Thank you for your patience.

Tapispisan-YvesJohn committed:
fix masonry layout

Villa-VillaDiana committed:
update login conditions

Villa-VillaDiana committed:
update table

Villa-VillaDiana committed:
update login condition

Villa-VillaDiana committed:
Merge pull request #15 from PUP-BSIT/main

pull changes from the main branch

Villa-VillaDiana committed:
update announcements table

MentokAlmera committed:
modify member page

MentokAlmera committed:
modify css

MentokAlmera committed:
add validators for student number

MentokAlmera committed:
add validator for birthday

Villa-VillaDiana committed:
add new tables and create adminseeder

Villa-VillaDiana committed:
add event state and status seeder

Villa-VillaDiana committed:
Merge pull request #18 from PUP-BSIT/backend

pull backend changes

Villa-VillaDiana committed:
fix indentations

MentokAlmera committed:
modify ts

Villa-VillaDiana committed:
Merge branch 'pages-members' into login-and-signup

Villa-VillaDiana committed:
Merge pull request #19 from PUP-BSIT/login-and-signup

pull login and signup changes

Villa-VillaDiana committed:
add backend functionality to view members and membership request

Villa-VillaDiana committed:
update backend to accept and decline members

Villa-VillaDiana committed:
update the remove member functionality

Villa-VillaDiana committed:
update the backend to create a member

Villa-VillaDiana committed:
Merge pull request #21 from PUP-BSIT/main

Villa-VillaDiana committed:
Merge branch 'freedom-wall' into login-and-signup

Villa-VillaDiana committed:
Merge pull request #22 from PUP-BSIT/login-and-signup

pull changes from login-and-signup

Villa-VillaDiana committed:
remove duplicate declarations in app.module

Villa-VillaDiana committed:
create FreedomWallController and update api.php

Villa-VillaDiana committed:
remove ng if and ng for, update data.service

Villa-VillaDiana committed:
add to do

Villa-VillaDiana committed:
update tables and add icons

create icons table
add icon_id in student table
 upload icons in assets\icons
update files in laravel_api\public

Villa-VillaDiana committed:
update img src, reupload missing icons

Tapispisan-YvesJohn committed:
fix use of vanilla js

Ternal-AprilJoy committed:
modify freedom wall modal.

Ternal-AprilJoy committed:
use form builder and add input validation.

Villa-VillaDiana committed:
Merge branch 'freedom-wall' into pages-members

Villa-VillaDiana committed:
Merge pull request #23 from PUP-BSIT/pages-members

pull the changes in backend

Villa-VillaDiana committed:
update freedomwall table and admin seeder

Villa-VillaDiana committed:
add backend functionality to post

Villa-VillaDiana committed:
add backend functionality to get and delete posts

Villa-VillaDiana committed:
update id format and add todo

Villa-VillaDiana committed:
add toastr alert and fix indentations

Villa-VillaDiana committed:
update the color of the toastr

Villa-VillaDiana committed:
fix indentation and spaces

Villa-VillaDiana committed:
update spaces

Ternal-AprilJoy committed:
remove styling in html.

Ternal-AprilJoy commented:
This pull request will be closed since there will be a new branch where these changes will be merged.
